<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddAzsDialog</class>
 <widget class="QDialog" name="AddAzsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>521</width>
    <height>248</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="3" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Телефон:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Расположение:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0" alignment="Qt::AlignTop">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Примечание:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="lineEditTerminalID"/>
   </item>
   <item row="5" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Reset|QDialogButtonBox::Save</set>
     </property>
     <property name="centerButtons">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="lineEditPhone"/>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEditAdress">
     <property name="cursorPosition">
      <number>0</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Терминал:</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QPlainTextEdit" name="plainTextEditComments"/>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelRegion">
     <property name="text">
      <string>Регион:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxRegion"/>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxRegion</tabstop>
  <tabstop>lineEditTerminalID</tabstop>
  <tabstop>lineEditAdress</tabstop>
  <tabstop>lineEditPhone</tabstop>
  <tabstop>plainTextEditComments</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
